main.go

```diff
@@ -151,7 +155,7 @@ number. This is a soft requirement and can be skipped by specifying the
 -f or --force flag.
 `,
 	Flags: []cli.Flag{
-		cli.BoolFlag{
+		&cli.BoolFlag{
 			Name:  "force,f",
```

Comment:
Apparently, this no longer works (tests are failing). It needs to be `&cli.BoolFlag{Name: "force", Aliases: []string{"f"}}`.

Reply:
We probably need to convert other short flags as well.

Reply:
Actually this was the only short flag.
The other aliases (such as `-f`) are subcommands and already use the `Aliases` way.